### PR TITLE
Documentation fixes

### PR DESCRIPTION
**Pull request by jsandham**

### CODE REVIEW

**Review by doctorcolinsmith — CHANGES_REQUESTED** [review]
Looks like there were changes in hipsparse.h that are unrelated to the documentation fix.

File: docs/source/conf.py [hunk 1]
```diff
@@ -48,32 +83,31 @@
...
-source_suffix = '.rst'
+source_suffix = ['.rst', '.md']
 
 # The master toctree document.
 master_doc = 'index'
 
 # General information about the project.
 project = u'hipSPARSE'
-copyright = u'2021, Advanced Micro Devices'
+copyright = u'2016-2022, Advanced Micro Devices'
```
Comment (doctorcolinsmith) [inline docs/source/conf.py:93]: It's 2023!

**Review by doctorcolinsmith — CHANGES_REQUESTED** [review]

File: docs/source/usermanual.rst [hunk 2]
```diff
@@ -1211,8 +1211,6 @@ hipsparseDestroy()
```
Comment (doctorcolinsmith) [inline docs/source/usermanual.rst:1211]: Don't forget to update the copyright at the top

**Review by doctorcolinsmith — CHANGES_REQUESTED** [review]

File: docs/source/conf.py (hunk 1 above)
Comment (jsandham) [reply docs/source/conf.py]: Fixed

File: docs/source/conf.py [hunk 3]
```diff
@@ -1,3 +1,23 @@
+"""Copyright (C) 2016-2022 Advanced Micro Devices, Inc. All rights reserved.
```
[the remark this answers is not in the document]
Comment (jsandham) [reply docs/source/conf.py]: Fixed

File: docs/source/usermanual.rst (hunk 2 above)
Comment (jsandham) [reply docs/source/usermanual.rst:1211]: This file does not contain a copyright as it is what is used in generating the documentation (I don't think any of the .rst files contain a copyright at the top in any of our libraries)

**Review by doctorcolinsmith — APPROVED** [review]

File: CMakeLists.txt [hunk 4]
```diff
@@ -1,5 +1,5 @@
 # ########################################################################
-# Copyright (C) 2018-2020 Advanced Micro Devices, Inc. All rights Reserved.
+# Copyright (C) 2018-2022 Advanced Micro Devices, Inc. All rights Reserved.
 #
```
Comment (ntrost57) [inline CMakeLists.txt:3]: 2023

**Review by ntrost57 — APPROVED** [review]

**Pull request merged**